src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1965,6 +1971,35 @@ public void cannotCreateRequestForClaimedReturnedItem(String requestType) {
     )));
   }
 
+  @Test
+  public void canFetchHundredRequests() {
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final List<IndividualResource> createdRequests =
```

Comment:
Personally, I would have a blank line between these two lines, to separate the multiple line initialisation out, and hopefully make it easier to read.